src/AdminConsole/appsettings.Development.json

```diff
@@ -11,9 +11,8 @@
     "CustomerAuthPlanName": "Professional Test"    
   },
   "ConnectionStrings": {
-    "sqlite": {
-      "admin": "Data Source=adminconsole_dev.db"
-    }
+    "sqlite:admin": "Data Source=adminconsole_dev.db",
+    "sqlite:audit": "Data Source=adminconsole_dev.db"
```

Comment:
@jrmccannon We removed the need for a separate audit db connection string, right?

Reply:
Correct. I'll get that cleaned up!

Reply:
Removed